mono/utils/atomic.h

```diff
@@ -464,4 +464,12 @@ extern void InterlockedWritePointer(volatile gpointer *dst, gpointer val);
 #define InterlockedAddP(p,add) InterlockedAdd64 ((volatile gint64*)p, (gint64)add)
 #endif
 
+/* The following functions cannot be found on any platform, and thus they can be declared without further existence checks */
+
+static inline gint32
+InterlockedSubtract (volatile gint32 *dest, gint32 sub)
+{
```

Comment:
Alright, I see! I will leave `atomics.h` as untouched as possible! :) Does it make sense to consider this replacement already when building/designing `unlocked.h` or is it too early?